### ISSUE TITLE
DatePicker dropdown doesn't have accessibility attributes

### ISSUE THREAD

**Issue by Ke1sy** [ant-design/ant-design#32825]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-6e93z?file=/index.css)

### Steps to reproduce
Expected sequence of steps:
1) Open the code sandbox link.
2) Open the calendar dropdown.
3) Explore the markup of the dropdown and find div with class 'ant-picker-dropdown'.

### What is expected?
DatePicker dropdown should have accessible attributes. Please see an example of accessible calendar made by W3 (https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/datepicker-dialog.html)
DatePicker dropdown should have accessible attributes:
•	role="dialog" 
•	aria-modal="true"
•	aria-label with selected month and year

Example: 
<div class="ant-picker-dropdown ant-picker-dropdown-placement-bottomLeft " role="dialog" aria-modal="true" aria-label="November 2021">

### What is actually happening?
Datepicker doesn't have any accessibility attributes:
<div class="ant-picker-dropdown ant-picker-dropdown-placement-bottomLeft " style="...">

| Environment | Info |
|---|---|
| antd | 4.16.3 |
| React | 16.14.0 |
| System | Windows 10 |
| Browser | Chrome 95 |

---
This issue is created in addition to the mentioned accessibility problems in DatePicker:
https://github.com/ant-design/ant-design/issues/32513

**Comment by afc163** [comment]
Duplicate of #32513
